src/static/css/most-viewed.css

```diff
@@ -0,0 +1,41 @@
+.headline-column--desktop {
```

Comment:
not sure if you want to go this route, but you can style the html (we really should be using json, as you say :) in emotion, e.g. this is how the article styles capi stuff https://github.com/guardian/guui/blob/master/src/pages/Article.js#L25-L44

Reply:
Oh great, that is so much nicer! Thanks